plugins/http_extract/src/http_extract/component.py

```
@@ -0,0 +1,580 @@
+
+        # Проверяем кэш
+        cache_key = self._generate_cache_key(url, params)
+        if self.config.cache_responses and cache_key in self._cache:
+            cache_entry = self._cache[cache_key]
+            if datetime.now(tz=UTC) < cache_entry["expires"]:
+                logger.debug(f"Using cached response for {url}")
+                return cache_entry["data"], cache_entry["headers"]
+
+        headers = {**self._auth_headers}
+        if self._session is None:
+            raise ValueError()
```

Ruff (0.12.2)

Unnecessary parentheses on raised exception

Remove unnecessary parentheses

(RSE102)